.env.example

```
@@ -108,5 +104,16 @@ DEV_AUTH_QF_ID=
 # optional, default: devadmin
 DEV_AUTH_USERNAME=devadmin
 
+# ─── Offline scripts (scripts/prewarm-graph.mjs, scripts/embed-corpus.mjs) ────
+# APP_URL: base URL of the running app that prewarm-graph.mjs calls into.
+# Default: http://localhost:3000
+APP_URL=
+# PREWARM_LIMIT: how many verses to process per prewarm-graph.mjs run. Default: 200
+PREWARM_LIMIT=
+# PREWARM_DELAY_MS: delay between prewarm-graph.mjs requests, in ms. Default: 250
+PREWARM_DELAY_MS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PREWARM_DELAY_MS key should go before the PREWARM_LIMIT key

(UnorderedKey)